Cargo.toml

```diff
@@ -35,8 +35,8 @@ dirs = "4.0"
 iocuddle = "0.1"
 serde-big-array = "0.4.1"
 kvm-ioctls = "0.11"
-static_assert_macro = "^1.1"
-bitfield = "^0.14"
+static_assertions = "^1.1.0"
+bitfield = "^0.13"
```

Comment:
Is there any particular reason we are downgrading this from `0.14.x` to `0.13.x`?

Reply:
We don't use anything from the `0.14` release, and reverting to `0.13` would make things simpler in downstream releases.